code/modules/paperwork/paper.dm

```diff
@@ -906,3 +906,8 @@
 
 	info = parsepencode(template, null, null, FALSE)
 #undef MAX_FIELDS
+
+/obj/item/paper/Almayer/CSG
```

Comment:
Not sure if there is anything in the style guide but type paths are usually all lower case

Reply:
yeah please use colonial_space_grunts or space_grunts or grunts_manual something similar

Reply:
You can remove the /Almayer too i think